### ISSUE TITLE
Slooooooow sync (

### ISSUE THREAD

**Issue by minzak** [byteball/headless-byteball#32]
I waste one week and i got 3.7m blockchain height.
Now when i check speed - it is near 20-30 seconds for 1 chain.
I nee more than 100 000 for fully sync.
P.S. i Have 2 core + 4Gb mem, but in top i see - all resources are free..
Why it is too slow?

P.S. in explorer still nothing!

```
root@byteball:/mnt# ./getinfo.sh
{
  "jsonrpc": "2.0",
  "result": {
    "last_mci": 3723438,
    "last_stable_mci": 3723421,
    "count_unhandled": 0
  },
  "id": 1
}

```
![1](https://user-images.githubusercontent.com/12154217/48055707-e494f700-e1b8-11e8-8105-50a0b17bfc82.PNG)

**Comment by tarmo888** [comment]
Currently https://explorer.byteball.org/ shows:
Main chain index: 3727516
Latest included mc index: 3727515

I am not sure that you are 100 000 behind. Maybe the resources are free because you have caught up already. Is the database around 30GB in size?

Join Slack for more support (#helpdesk and #tech channels) https://slack.byteball.org/

**Comment by minzak** [comment]
H'm yes. you right.
I'm mistake because always saw to highest value - "level: 3840701" )))
Thanks.
